src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -767,9 +767,14 @@ public String getCsvDetailedResponsesRow(FeedbackSessionResultsBundle fsrBundle,
         FeedbackRubricResponseDetails frd = (FeedbackRubricResponseDetails) feedbackResponseAttributes.getResponseDetails();
         StringBuilder detailedResponsesRow = new StringBuilder(100);
         for (int i = 0; i < frd.answer.size(); i++) {
+
+            int chosenIndex = frd.answer.get(i);
+            if (chosenIndex == -1 && !isMissingResponsesShown) {
+                continue;
+            }
```

Comment:
See line 777. If the unanswered subquestion is the first subquestion, any comments that should have shown up will not appear on the csv.